src/main/resources/assets/securitycraft/lang/es_es.json

```diff
@@ -0,0 +1,768 @@
+    "gui.securitycraft:scManual.options": "Opciones disponibles:",
+    "gui.securitycraft:scManual.modules": "Módulos disponibles:",
+	"gui.securitycraft:scManual.recipe.keypad": "Coloca un marco y haz clic con un panel numérico.",
+	"gui.securitycraft:scManual.recipe.keypad_chest": "Coloca un cofre y haz clic derecho con el panel numérico.",
+	"gui.securitycraft:scManual.recipe.keypad_furnace": "Coloca un horno y haz clic derecho con el panel numérico.",
+	"gui.securitycraft:scManual.recipe.reinforced": "Los bloques reforzados se pueden fabricar de la siguiente manera, si no se especifica lo contrario en el libro. Coloca el equivalente estándar (de la piedra reforzada, sería la piedra) y rómpelo con el reforzador de bloques universal o haz clic derecho con el reforzador de bloques universal, colócalo en el hueco y cierra la interfaz.",
+	"gui.securitycraft:scManual.recipe.reinforced_iron_trapdoor": "Los bloques reforzados se pueden fabricar de la siguiente manera, si no se especifica lo contrario en el libro. Coloca el equivalente estándar (de la piedra reforzada, sería la piedra) y rómpelo con el reforzador de bloques universal o haz clic derecho con el reforzador de bloques universal, colócalo en el hueco y cierra la interfaz.",
+	"gui.securitycraft:scManual.recipe.adminTool": "La herramienta de administrador solo está disponible en modo creativo. Debe activarse en el archivo de configuración.",
+	"gui.securitycraft:scManual.recipe.admin_tool": "La herramienta de administrador solo está disponible en modo creativo. Debe activarse en el archivo de configuración.",
+	"gui.securitycraft:scManual.recipe.reinforced_crystal_quartz": "Fabricado con bloques de cuarzo cristal normales con un reforzador de bloques universal.",
+	"gui.securitycraft:scManual.recipe.sc_manual": "El manual de SecurityCraft solo está disponible en el inventario creativo o escribiendo «/sc help».",
+	"gui.securitycraft:scManual.reinforced": "Bloques reforzados",
+	"gui.securitycraft:scManual.designedBy": "Diseñado por: %s",
+	"gui.securitycraft:ims.hostileAndPlayers": "Jugadores y criaturas hostiles",
+	"gui.securitycraft:ims.hostile": "Criaturas hostiles",
+	"gui.securitycraft:ims.target": "Objetivo:",
+	"gui.securitycraft:invScan.emitRedstone": "Emitir redstone",
+	"gui.securitycraft:invScan.checkInv": "Comprobar inventario",
+	"gui.securitycraft:invScan.explanation.1": "Este ajuste comprobará el invent.",
+	"gui.securitycraft:invScan.explanation.2": "del jugador. Si tiene algún objeto",
+	"gui.securitycraft:invScan.explanation.checkInv.3": "prohibido, se eliminará",
+	"gui.securitycraft:invScan.explanation.checkInv.4": "de su inventario.",
+	"gui.securitycraft:invScan.explanation.emitRedstone.3": "prohibido, se emitirá una",
+	"gui.securitycraft:invScan.explanation.emitRedstone.4": "señal durante 3 segundos.",
```

Comment:
This really is an abomination. We should probably properly split the text instead of using multiple lines. Then again, the GUI is set up so that the button perfectly fits into the text...

Reply:
Now it would fit without problems. If we add «señal de redstone» it would be so much long, but we could try shortening «segundos» into «seg.» to keep it fit.

Reply:
I maybe modify a bit the strings to keep all fitted, in this and the previous string.

Reply:
It will appear as:

	Este ajuste escaneará el invent.
	del jugador. Si tiene algún
	objeto prohibido, se emitirá una
	señal de redstone 3 segundos.